backend/src/services/integration-auth/integration-app-list.ts

```diff
@@ -455,6 +455,28 @@ const getAppsCircleCI = async ({ accessToken }: { accessToken: string }) => {
   return apps;
 };
 
+/**
+ * Return list of projects for Databricks integration
+ */
+const getAppsDatabricks = async ({ accessToken }: { accessToken: string }) => {
+  const res = await request.get<{ scopes: { name: string; backend_type: string }[] }>(
+    `${IntegrationUrls.DATABRICKS_API_URL}/2.0/secrets/scopes/list`,
+    {
+      headers: {
+        Authorization: `Bearer ${accessToken}`,
+        "Accept-Encoding": "application/json"
+      }
+    }
+  );
+
+  const scopes = res.data.scopes.map((a) => ({
+    name: a.name, // name maps to unique scope name in Databricks
+    backend_type: a.backend_type
+  }));
```

Comment:
If a user doesn't have any scopes this throws an error - might be nice to return an empty array and notify the user they have no scopes

Reply:
great catch!